readthedocs/proxito/views/serve.py

```diff
@@ -389,9 +389,14 @@ def get(self, request, project):
         )
         path = build_media_storage.join(storage_path, 'robots.txt')
 
+        log.bind(
+            project_slug=project.slug,
+            version_slug=version.slug,
+        )
```

Comment:
Does separating the `bind` and `info` give much value here? Are we just doing it to be consistent?

Reply:
In this particular case, it does not, since we are calling `log.info` just once. However, if we add another line later, we don't need to repeat `project_slug` and `version_slug`.